src/libsyntax/parse/token.rs

```diff
@@ -73,6 +74,7 @@ impl Lit {
         match *self {
             Byte(_) => "byte literal",
             Char(_) => "char literal",
+            Err(_) => "error literal",
```

Comment:
invalid literal

Reply:
Uh, what else do I have to do?

Reply:
Sorry, should have been clearer: Change the description from `"error literal"` to`"invalid literal"`.

Reply:
Done.